front/lib/api/assistant/conversation.ts

```diff
@@ -154,18 +158,22 @@ export async function updateConversation(
     throw new Error(`Conversation ${conversationId} not found`);
```

Comment:
This one is supposedly not expcted so it can throw but the one below with canAccessConversation is actually possible if I'm loading a conversation I used to have access to. So it should not throw should it?